samples/subsys/usb/hid-cdc/src/main.c

```diff
@@ -620,6 +626,13 @@ void main(void)
 
 	usb_hid_register_device(hid1_dev, hid_kbd_report_desc,
 				sizeof(hid_kbd_report_desc), &ops);
+
+	ret = usb_enable(status_cb);
+	if (ret != 0) {
```

Comment:
`if (usb_enable(status_cb))`?

Reply:
Personal opinion: yeah, it could be[*], but that would make it slightly more inconvenient to add diagnostics to display exactly why it failed when debugging or maintaining.

[*] well, actually this:

```
if (usb_enable(status_cb) != 0) {
```

since we're courting MISRA these days and treating `int` values as booleans is not acceptable.